test/verify/machineslib.py

```diff
@@ -283,7 +283,7 @@ def prepareStorageDeviceOnISCSI(self, target_iqn):
                   targetcli /iscsi/%(tgt)s/tpg1/acls create %(ini)s
                   """ % {"tgt": target_iqn, "ini": orig_iqn})
 
-        self.addCleanup(m.execute, "targetcli /backstores/ramdisk delete test && targetcli /iscsi delete %s" % target_iqn)
+        self.addCleanup(m.execute, "targetcli /backstores/ramdisk delete test && targetcli /iscsi delete %s && iscsiadm -m node -o delete" % target_iqn)
```

Comment:
This is not related to this commit. What are you trying to do though?Delete the target? looks like you need to specify the targetname for that as well.

Reply:
Oh, I just try to delete all records in the node mode of iscsiadm. Sometimes, I found that, when we try to create an ISCSI and the iqn is already in the 'iscsiadm -m node', then there will be an error raised even though there is no such session with the iqn.